README.md

```diff
@@ -313,6 +313,85 @@ The default temp dir used is  'os.TempDir()'. Use the following API to set a new
     apiKey, err := rtManager.RegenerateAPIKey()
 ```
 
+## Distribution APIs
+### Creating a Service Manager
+#### Creating Distribution Details
```

Comment:
Some of the APIs have return values (error). Let's reflect this in the documentation, as it is done similarly for other APIs in the README. This may same errors for the people who use these APIs.